cupy/cuda/nvtx.pyx

```diff
@@ -170,3 +173,22 @@ cpdef void RangePop() except *:
     pair of ``RangePush*()`` to ``RangePop()`` calls.
     """
     nvtxRangePop()
+
+
+cpdef unsigned long long RangeStartC(str message, uint32_t color=0) except *:
```

Comment:
I think `RangeStart` is better function name.
Currently, There are `RangePushC`. But, this method name is confusing.